setup.py

```diff
@@ -73,12 +87,23 @@
         'scripts/patch_recursive_apply.py',
         'scripts/qskope.py'],
     author="Niftools Developers",
+    cmdclass=cmdclass,
+    # these are optional and override conf.py settings
+    command_options={
+        'build_docs': {
+            'project': ('setup.py', "PyFFI"),
+            'version': ('setup.py', version),
+            'release': ('setup.py', version),
+            'source_dir': ('setup.py', 'docs/')
+        }
+    },
     license="BSD",
     keywords="fileformat nif cgf binary interface stripify",
     platforms=["any"],
     description="Processing block structured binary files.",
     classifiers=[_f for _f in classifiers.split("\n") if _f],
     long_description=long_description,
     url="https://github.com/niftools/pyffi",
-    download_url="https://github.com/niftools/pyffi/releases"
+    download_url="https://github.com/niftools/pyffi/releases",
+    install_requires=requirements
```

Comment:
Should not use requirement-dev as the requirements - https://caremad.io/posts/2013/07/setup-vs-requirement/

Reply:
Fixed